cg/meta/workflow/raredisease.py

```diff
@@ -274,3 +282,20 @@ def parse_analysis(self, qc_metrics_raw: list[MetricsBase], **kwargs) -> Nextflo
         return super().parse_analysis(
             qc_metrics_raw=qc_metrics_raw, qc_metrics_model=qc_metrics_model, **kwargs
         )
+
+    def get_sample_name_mapping_csv_path(self, case: str) -> Path:
+        """Return the path to the CSV file containing the mapping between sample names and internal ids."""
+        return Path(self.get_case_path(case), f"{case}_customer_internal_mapping.csv")
+
+    def export_customer_internal_mapping_csv(self, case: str, output_path: Path):
+        """Export a CSV file mapping customer sample names to internal sample IDs."""
+        LOG.info(f"Exporting customer internal mapping CSV for case {case} to {output_path}")
+        with output_path.open("w", newline="") as csvfile:
+            writer = csv.writer(csvfile)
+            writer.writerow(
+                ["customer_id", "internal_id"]
```

Comment:
I would still prefer that the first column would be "custome_sample_name" but I understand that this is something hard-coded in raredisease and there is little we can do about it now. Maybe for a future release we could have this changed? 😬